tidy-data/01-Reshape-Data/01-Reshape-Data.Rmd

```diff
@@ -269,46 +269,50 @@ question("What are the variables in cases?",
 
 ![](https://vimeo.com/229581273)
 
-### gather()
+### pivot_longer()
 
-You can use the `gather()` function in the **tidyr** package to convert wide data to long data. Notice that `gather()` returns a tidy copy of the dataset, but does not alter the original dataset. If you wish to use this copy later, you'll need to save it somewhere.
+You can use the `pivot_longer()` function in the **tidyr** package to convert wide data to long data. Notice that `pivot_longer()` returns a tidy copy of the dataset, but does not alter the original dataset. If you wish to use this copy later, you'll need to save it somewhere.
 
 ```{r echo = TRUE}
-cases %>% gather(key = "year", value = "n", 2, 3, 4)
+cases %>% 
+  pivot_longer(
+    cols = -Country, 
+    names_to = "year", 
+    values_to = "n"
+    )
 ```
 
-Let's take a closer look at the `gather()` syntax.
+Let's take a closer look at the `pivot_longer()` syntax.
 
-### gather() syntax
+### pivot_longer() syntax
 
 Here's the same call written without the pipe operator, which makes the syntax easier to see.
 
 ```{r echo = TRUE, eval = FALSE}
-gather(cases, key = "year", value = "n", 2, 3, 4)
+pivot_longer(cases, cols = c(`2011`, `2012`, `2013`), names_to = "year", values_to = "n")
```

Comment:
The `cols` argument here lists the actual years, whereas the previous chunk uses `-Country` to select columns. Should they be consistent? The text reads that it is the same call written without the pipe operator, so it might be simpler to only have the inclusion of `cases %>%` be the thing that is different here.

Reply:
Just got down the "specifying columns" section- perhaps suggests we should keep this example as is with the years named....

Reply:
Agreed + fixed